pkg/remote/aws/route53_record_detail_fetcher.go

```diff
@@ -0,0 +1,36 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+)
+
+type Route53RecordDetailsFetcher struct {
+	reader       terraform.ResourceReader
+	deserializer *resource.Deserializer
+}
+
+func NewRoute53RecordDetailsFetcher(provider terraform.ResourceReader, deserializer *resource.Deserializer) *Route53RecordDetailsFetcher {
+	return &Route53RecordDetailsFetcher{
+		reader:       provider,
+		deserializer: deserializer,
+	}
+}
+
+func (r *Route53RecordDetailsFetcher) ReadDetails(res resource.Resource) (resource.Resource, error) {
+	ctyVal, err := r.reader.ReadResource(terraform.ReadResourceArgs{
+		Ty:         aws.AwsRoute53RecordResourceType,
+		ID:         res.TerraformId(),
+		Attributes: map[string]string{},
```

Comment:
same as in the other pr you don't need the empty map an should use GenericDetailsFetcher